app/javascript/react/components/bands/BandTile.js

```diff
@@ -3,8 +3,9 @@ import { Link } from 'react-router-dom'
 
 const BandTile = (props) => {
   const {band} = props
+
   return (
-    <div>
+    <div style={{backgroundImage: `url(${band.image.url})`, backgroundSize: "cover", backgroundPosition: "center"}}>
```

Comment:
ill allow this but only because background images are frustration. the size and position should ideally be designated in your CSS file instead